internal/staticanalysis/parsing/js/js_parsing_test.go

```diff
@@ -299,11 +298,18 @@ var index = 0,
 
 func TestParseJS(t *testing.T) {
 	const printAllJson = false
+	const jsParserDir = "/tmp/package-analysis-js-parsing-test-parser"
```

Comment:
You probably want to use `t.TempDir()` here - it is unique per test invocation, and is automatically cleaned up by Go.